### PR TITLE
Show all 3D tile content tests under correct heading

### PR DESCRIPTION
**Pull request by hpinkos**
@bagnell

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@hpinkos, thanks for the pull request! Maintainers, we have a signed CLA from @hpinkos, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Specs/Scene/Batched3DModel3DTileContentSpec.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ defineSuite([
         'Core/Transforms',
         'Specs/Cesium3DTilesTester',
         'Specs/createScene'
-    ], function(
+    ], 'Scene/Batched3DModel3DTileContent', function(
```
Comment (bagnell) [inline Specs/Scene/Batched3DModel3DTileContentSpec.js]: This one is already the first require in the file.

**Comment by hpinkos** [comment]
Whoops, good catch.  Ready @bagnell!

File: Specs/Scene/Composite3DTileContentSpec.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ defineSuite([
         'Core/HeadingPitchRange',
         'Specs/Cesium3DTilesTester',
         'Specs/createScene'
-    ], function(
+    ], 'Scene/Composite3DTileContentSpec', function(
```
Comment (bagnell) [inline Specs/Scene/Composite3DTileContentSpec.js]: Don't add `Spec` to the end of the name.

File: Specs/Scene/Tileset3DTileContentSpec.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ defineSuite([
         'Core/HeadingPitchRange',
         'Specs/Cesium3DTilesTester',
         'Specs/createScene'
-    ], function(
+    ], 'Scene/Tileset3DTileContentSpec', function(
```
Comment (bagnell) [inline Specs/Scene/Tileset3DTileContentSpec.js]: And here.

**Comment by hpinkos** [comment]
Gah, sorry @bagnell.  This should be good now

**Pull request merged**